photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSource.java

```diff
@@ -88,4 +88,9 @@ public FPSRatedVideoMode(
     public boolean isVendorCamera() {
         return ConfigManager.getInstance().getConfig().getHardwareConfig().hasPresetFOV();
     }
+
+    @Override
+    public boolean hasLEDs() {
```

Comment:
Unrelated: we need to refactor how we deal with LEDs at some point

Reply:
yea, maybe fire up another discord discussion on it. Really at the core.... in 2024+, are "LED's around the camera" still a thing?